src/controllers/user.controller.ts

```diff
@@ -39,16 +39,16 @@ export class UserController {
     public recommender: RecommenderService,
     @inject.setter(AuthenticationBindings.CURRENT_USER)
     public setCurrentUser: Setter<UserProfile>,
-    @inject(OtherServicesBindings.HASH_PASSWORD)
-    public hashPassword: HashPassword,
+    @inject(PasswordHasherBindings.PASSWORD_HASHER)
+    public passwordHahser: PasswordHasher,
     @inject(JWTAuthenticationBindings.SERVICE)
-    public jwt_authentication_service: JWTAuthenticationService,
+    public jwtAuthenticationService: JWTAuthenticationService,
   ) {}
 
   @post('/users')
   async create(@requestBody() user: User): Promise<User> {
     validateCredentials(_.pick(user, ['email', 'password']));
-    user.password = await this.hashPassword(user.password, 10);
```

Comment:
Do we want to set the rounds binding key to `10` for this change?

Reply:
@bajtos This `rounds` is set at the application level as 10 :) see https://github.com/strongloop/loopback4-example-shopping/pull/51/files#diff-8f42631f8d7d8cbaf47074d3ea5536d1R65